scattering/scattering.py

```diff
@@ -100,6 +100,91 @@ def structure_factor(trj, Q_range=(0.5, 50), n_points=1000, framewise_rdf=False)
         S[i] = num/denom
     return Q, S
 
+
+def structure_factor_pair(trj, pair, Q_range=(0.5, 50), n_points=1000, framewise_rdf=False, method='fz'):
+    rho = np.mean(trj.n_atoms / trj.unitcell_volumes)
```

Comment:
Please see the documentation for docstrings: https://www.python.org/dev/peps/pep-0257/.  Each function should have a docstring that has a short description of what the function does, the list of arguments, and what the function returns.  See the docstring in `structure_factor()` above.